dnf-behave-tests/dnf/plugins-core/config-manager-addrepo.feature

```diff
@@ -0,0 +1,437 @@
+@dnf5
+Feature: dnf "config-manager" command - test "addrepo" subcommand
+
+
+Background:
+  Given I enable plugin "config_manager"
+    And I create file "/etc/yum.repos.d/repo1.repo" with
+        """
+        [repo1]
+        name=repo1 test repository
+        enabled=1
+        baseurl=http://something1.com/os/
+        """
+    And I create file "/etc/yum.repos.d/repo2.repo" with
+        """
+        [repo2]
+        name=repo2 test repository
+        enabled=0
+        baseurl=http://something2.com/os/
+        """
+
+
+Scenario: test "addrepo" from "baseurl", "enabled=1" is set by default
+   When I execute dnf with args "config-manager addrepo --set=baseurl=http://something.com/os/"
+   Then the exit code is 0
+    And file "/etc/yum.repos.d/something.com_os_.repo" contents is
+        """
+        [something.com_os_]
+        name=created by dnf5 config-manager
```

Comment:
This is not a problem. What about to add RepoId also to name (combine RepoID with the current content of repo name). It would create a name unique and it will be more like a name then a comment how the name was created.